config/checkstyle/tree_walker_module_directives.xml

```diff
@@ -44,9 +44,9 @@
 </module>
 <module name="MissingSwitchDefault"/>
 <module name="MultipleStringLiterals">
-    <property name="allowedDuplicates" value="3"/>
+    <property name="allowedDuplicates" value="5"/>
```

Comment:
Hmm I'd rather leave it at 3 TBH...

Reply:
But I have 5 `spaces` is this file  :/

Reply:
Ok but this limit doesn't matter for a space once it's included in the allowed regexes below :thinking:

Reply:
aaaand we don't want to increase the limit to 5 for every other (non-whitelisted) string

Reply:
Yeah, you are right. My misunderstood.